modules/k8s/Makefile.tanka

```diff
@@ -36,11 +42,11 @@ k8s/tanka/fmt-test: satoshi/check-deps
 	tk fmt --test .
 
 ## Generate manifests using tanka
-k8s/tanka/generate: satoshi/check-deps jsonnet/install k8s/tanka/vendor/charts
+k8s/tanka/generate: satoshi/check-deps jsonnet/install k8s/tanka/charts/vendor k8s/tanka/charts/patch
 	@@${BUILD_HARNESS_EXTENSIONS_PATH}/modules/k8s/tanka/generate.sh $(GENERATE_ARGS)
 
 ## Generate manifests of specific app using tanka
-k8s/tanka/generate/%: satoshi/check-deps jsonnet/install k8s/tanka/vendor/charts
+k8s/tanka/generate/%: satoshi/check-deps jsonnet/install k8s/tanka/charts/vendor
```

Comment:
This fails for most repos as `./charts/patches` does not exist.

I think this should probably look for a `patches` directory at the root (outside of charts) as `charts` is managed by tanka.

It would also need to check for the iexistance of a `patches` directory before attempting to find files under it.